packages/nodejs/.changesets/appsignal-client-is-available-in-global-object.md

```diff
@@ -0,0 +1,5 @@
+---
+bump: "patch"
+---
+
+AppSignal's client is now available in global object as: `__APPSIGNAL__`
```

Comment:
Do we want this to be part of the documented public interface? If we don't want people to rely on this, then we probably shouldn't mention it in the changelog.

Reply:
I'm not sure, I don't see it as a bad thing

Reply:
I don't think we want to advertise where we store it. Users shouldn't access it directly like `global.__APPSIGNAL__`. If we want to allow access it should be a helper function like `Appsignal.config()` to fetch it from the config object.

We should make a helper function for that, but users don't need to know about it at the moment.

```suggestion
The AppSignal client is stored on the global object as to make access easier for AppSignal developers.
```

Reply:
OK, as the issue states, will add the accessor on a future PR